mlvm/test_mlvm.py

```diff
@@ -0,0 +1,259 @@
+import os
+
+import pkg_resources
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+class MLVMTestCase(DataprocTestCase):
```

Comment:
There seems to be too many test cases here - 26

May we test all the functionality on the single cluster that has all features/packages instead? It will reduce test time and amount of test code.

Reply:
I was able to reasonably reduce this to 4. Verifying locally.